internal/cmdrender/executor.go

```diff
@@ -253,8 +254,10 @@ func hydrate(ctx context.Context, pn *pkgNode, hctx *hydrationContext) (output [
 }
 
 // runPipeline runs the pipeline defined at current pkgNode on given input resources.
-func (pn *pkgNode) runPipeline(ctx context.Context, input []*yaml.RNode) ([]*yaml.RNode, error) {
+func (pn *pkgNode) runPipeline(ctx context.Context, input []*yaml.RNode, truncateOutput bool) ([]*yaml.RNode, error) {
 	const op errors.Op = "pipeline.run"
+	pr := printer.FromContextOrDie(ctx)
+	pr.Printf("Package %q:\n", pn.pkg.DisplayPath)
```

Comment:
I think there is a function on the printer specifically for printing package information.

Reply:
Yeah, we can just use `pr.PkgPrintf("")` here ?

Reply:
Will do it.

Reply:
Done